cupy/core/elementwise.pxi

```diff
@@ -468,7 +468,7 @@ cdef class ElementwiseKernel:
             (i.e., the arrays are reshaped without copy to the minimum
             dimension) by default. It may make the kernel fast by reducing the
             index calculations.
-        options (list): Options passed to the ``nvcc`` command.
+        options (tuple): Options passed to the ``nvcc`` command.
```

Comment:
Ah, we are no longer using ``nvcc``.
Let's fix it together and add link to https://docs.nvidia.com/cuda/nvrtc/index.html#group__options